test/integration/standalone_single_project/standalone_single_project_test.go

```diff
@@ -39,9 +39,19 @@ func TestStandaloneSingleProjectExample(t *testing.T) {
 	// define and write a custom verifier for this test case call the default verify for confirming no additional changes
 	standaloneSingleProjT.DefineVerify(func(assert *assert.Assertions) {
 		// perform default verification ensuring Terraform reports no additional changes on an applied blueprint
-		standaloneSingleProjT.DefaultVerify(assert)
```

Comment:
Are we seeing drift?

Reply:
Got a cycle error:

> Error: Invalid for_each argument
        	            	
        	            	  on .terraform/modules/cicd.project-iam-bindings/modules/projects_iam/main.tf line 52, in resource "google_project_iam_member" "project_iam_additive":
        	            	  52:   for_each = module.helper.set_additive
        	            	    ├────────────────
        	            	    │ module.helper.set_additive is a set of dynamic, known only after apply
        	            	
        	            	The "for_each" set includes values derived from resource attributes that
        	            	cannot be determined until apply, and so Terraform cannot determine the full
        	            	set of keys that will identify the instances of this resource.
        	            	
        	            	When working with unknown values in for_each, it's better to use a map value
        	            	where the keys are defined statically in your configuration and where only
        	            	the values contain apply-time results.
        	            	
        	            	Alternatively, you could use the -target planning option to first apply only
        	            	the resources that the for_each value depends on, and then apply a second
        	            	time to fully converge.